Source/LeapMotion/Private/LeapWrapper.cpp

```diff
@@ -155,14 +168,16 @@ LEAP_TRACKING_EVENT* FLeapWrapper::GetInterpolatedFrameAtTime(int64 TimeStamp)
 		LeapInterpolateFrame(ConnectionHandle, TimeStamp, InterpolatedFrame, InterpolatedFrameSize);
 	}
 
-	return InterpolatedFrame;
+	//Disable interpolation test
```

Comment:
@lazloringUH if we are not returning the interpolated frame then most of this code (or method) no longer makes sense?